StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -170,14 +216,16 @@ extension STPAPIClient: FinancialConnectionsAPIClient {
         }
         return self.get(
             resource: APIEndpointListAccounts,
-            parameters: parameters
+            parameters: parameters,
+            canUseConsumerPublishableKey: false
         )
     }
 
     func fetchFinancialConnectionsSession(clientSecret: String) -> Promise<StripeAPI.FinancialConnectionsSession> {
         return self.get(
             resource: APIEndpointSessionReceipt,
-            parameters: ["client_secret": clientSecret]
+            parameters: ["client_secret": clientSecret],
+            canUseConsumerPublishableKey: false
```

Comment:
at first I was wondering whether we can avoid scattering `canUseConsumerPublishableKey` everywhere, but its likely a good idea to "force the developer to think" (...on whether this should be used or not) 👍 

that also goes to the note about commenting: maybe we should leave some basic guidelines/explanation of when one would want to set this or not (just some general guidance so a new dev is not completely confused what this param is as it is a bit ambigious)

Reply:
Will add some docs on this!